navigator_api/main.py

```diff
@@ -70,58 +71,40 @@ def workflow_state(dataset_id):
         "id": "xxx-yyy-zzz",
         "milestones": [
             {
-                "id": "xxx",
-                "title": "Naomi Data Prep",
+                "id": mockup.id(),
+                "title": mockup.milestone(),
```

Comment:
This is overthinking IMO. We should keep static/hardcoded content here. I don't see any value in introducing the `utils.mockup` module. Those stubs will go away in a couple of days after nav_engine gets a working API.